test/unit/test_power_curve_toolkit.py

```diff
@@ -16,13 +16,35 @@ def setUp(self):
         self.x = pd.Series(np.random.random(100) * 30)
         self.y = pd.Series(logistic5param(self.x, *params) + np.random.random(100) * noise)
 
+        # power curve source: https://github.com/NREL/turbine-models/blob/master/Offshore/2020ATB_NREL_Reference_15MW_240.csv
+        self.nrel_15mw_wind = pd.Series(np.arange(4, 26))
+        self.nrel_15mw_power = pd.Series(np.array([720, 1239, 2271, 3817, 5876, 8450, 11536, 1500, 1500, 1500, 1500, 1500, 1500, 1500, 1500, 1500, 1500, 1500, 1500, 1500, 1500, 1500]))
```

Comment:
Looks like an extra 0 is missing from the rated power (should be 15000)

Reply:
Good catch, that's an embarrassing copy paste error.